build/depends.py

```diff
@@ -1260,7 +1260,7 @@ def configure(self, build, conf):
             # that we include. We can't define QT_NO_DEBUG because that would
             # mean turning off QDebug output. qt_noop() is what Qt defines
             # Q_ASSERT to be when QT_NO_DEBUG is defined.
```

Comment:
If we're not using `qt_noop()` anymore, the comment should be edited.

Reply:
I can do that but what about failed autotests?

Reply:
@sblaisot do you know why AppVeyor is failing?